src/components/Card/index.jsx

```diff
@@ -1,30 +1,92 @@
 import React, { Component } from 'react';
 import './index.css';
 
+import {BsPencil, BsX, BsCheck} from 'react-icons/bs';
+
 class Card extends Component {
+
     state = {
-        isChecked: false
+        isChecked: false,
+        isEdit: false,
+        cardData : {
+            Logo: 'Caption',
+            Text: 'Some text here...',
+        },
```

Comment:
Still, naming convention says that objects, variables, constants, properties, etc. should be named in **camelCase.**